README.md

```diff
@@ -39,27 +32,48 @@ The browser will open and you should see this text : "Hello webpack!", if this i
 http://localhost:3000
 ```
 
+## Built With
 
-- 6
+- HTML
+- HTML Webpack Plugin
+- CSS
+- JavaScript / ES6
+- Webpack
 
- You can start editing the files in the "src" folder
+### Install
 
-## Happy coding! 🌈
+Clone the repository
+https://github.com/Kayonga99/ToDo-List
+npm . init -y
+npm install
 
-if you are interested in learning more about webpack click the link below
+## Run tests
 
-[click here!](https://webpack.js.org/guides/getting-started/)
+npx stylelint "\*_/_.{css,scss}"
+npx hint .
+npx eslint .
 
-## Show your support
+- Web Browser
+- [Visual Studio code](https://code.visualstudio.com/)
+[Live demo]()
```

Comment:
- [ ] [OPTIONAL] Please, consider fixing the broken live demo link or just delete this link on current milestone 👍